components/support/support-additional.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "id": 20000,
```

Comment:
Where do the ids come from? Is there any chance these could clash in the future?